content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
+
+**Author:** Shane Utt (Kong)
+
+We're pleased to announce that key Gateway API resources are graduating to
+beta with the `v0.5.0` release of the API. In addition to `GatewayClass`,
+`Gateway` and `HTTPRoute` graduating to beta, this release introduces several
+new concepts to the API that we'll cover below.
+
+## What is Gateway API
+
+Gateway API is a collection of resources centered around `Gateway` resources
+(which represent the underlying network gateways / proxy servers) to enable
+robust Kubernetes service networking through expressive, extensible and
+role-oriented interfaces that are implemented by many vendors and have broad
+industry support.
+
+Originally conceived as a successor to the well known `Ingress` API, the
```

Comment:
I'd make Ingress be a hyperlink to https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/

Reply:
```suggestion
Originally conceived as a successor to the well known [Ingress](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/) API, the
```